test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -303,15 +305,13 @@ std::string RandomCode::generate(int _maxOpNumber, RandomCodeOptions _options)
 	return code;
 }
 
-std::string RandomCode::randomUniIntHex(u256 _maxVal)
+std::string RandomCode::randomUniIntHex(u256 const& _minVal, u256 _maxVal)
 {
 	if (_maxVal == 0)
 		_maxVal = std::numeric_limits<uint64_t>::max();
 	refreshSeed();
-	int rand = randUniIntGen() % 100;
-	if (rand < 50)
-		return toCompactHexPrefixed((u256)randUniIntGen() % _maxVal);
-	return toCompactHexPrefixed((u256)randUInt64Gen() % _maxVal);
+	u256 value = _minVal + (u256)randUInt64Gen() % (_maxVal - _minVal);
```

Comment:
Now it's in `[_minVal; _maxVal-1]`, not what I would expect from function signature like this